# bash's own private copy of libreadline.
# It needs some header file help for handling Unicode and/or
# multi-byte encodings.
# This is Solaris-specific.
--- lib/readline/rlprivate.h	2009-01-22 21:56:49.000000000 -0500
+++ lib/readline/rlprivate.h	2009-08-16 10:55:56.178609000 -0400
@@ -23,6 +23,18 @@
 #if !defined (_RL_PRIVATE_H_)
 #define _RL_PRIVATE_H_
 
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#if defined(HANDLE_MULTIBYTE)
+
+#include <limits.h>
+#include <wchar.h>
+#include <wctype.h>
+
+#endif /* HANDLE_MULTIBYTE */
+
 #include "rlconf.h"	/* for VISIBLE_STATS */
 #include "rlstdc.h"
 #include "posixjmp.h"	/* defines procenv_t */
